### DISCUSSION TITLE
Virtfusion resource pack support

### DISCUSSION THREAD

**Discussion by Pachini** [Paymenter/Paymenter#382] [Ideas]
Hi!

There is already integration with virtfusion, thanks! But also it would be very cool if you can implement "resource pack" feature, all the api on the virfusion side are [already there](https://docs.virtfusion.com/guides/self-service#resource-packs )

**Comment by CorwinDev** [comment]
Hi, I already had this on my todo, so this will be added shortly :)

**Reply by Pachini** [reply]
Awesome! Thank you
